test/quantization/test_numeric_suite.py

```diff
@@ -0,0 +1,24 @@
+from torch.quantization import default_eval_fn, quantize
+from torch.quantization._numeric_suite import compare_weights
+from torch.testing._internal.common_quantization import (
+    AnnotatedConvModel,
+    QuantizationTestCase,
+)
+
+
+class EagerModeNumericSuiteTest(QuantizationTestCase):
+    def test_compare_weights(self):
+        r"""Compare the weights of float and quantized conv layer
+        """
+        # eager mode
+        annotated_conv_model = AnnotatedConvModel().eval()
+        quantized_annotated_conv_model = quantize(
+            annotated_conv_model, default_eval_fn, self.img_data
+        )
+        weight_dict = compare_weights(
+            annotated_conv_model.state_dict(),
+            quantized_annotated_conv_model.state_dict(),
+        )
```

Comment:
Can you add a test for a simpler model, let say
``` m=nn.Conv2d(...)
 ```
Will the code work in that case? I am not sure because find_nearest_match does this:
```
        pattern2 = "".join(s2.split(".")[0:-2])
```

Reply:
Seems the case you mentioned didn't show properly, could you post it again? @raghuramank100

Reply:
I tried it out and it's working. If we do .join(str[0:-2]) we will get empty string.